test/Common/ProductColumnTypes.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.Azure.WebJobs.Extensions.Sql.Tests.Common
 {
     public class ProductColumnTypes
     {
-        public int ProductID { get; set; }
+        public int ProductId { get; set; }
```

Comment:
I have these duplicated in test-outofproc folder as well, probably a good idea to move them to a common project and reference them to avoid these redundant issues, but for now can you please update them there as well, thanks!